public/linux/com.github.mbrlabs.Lorien.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Type=Application
+Name=Lorien
+GenericName=Note Taking
+Comment=Infinite canvas drawing/whiteboarding app. Made with Godot.
+Comment[de]=Unbegrenzte Zeichen- und Whiteboard-App. Erstellt mit Godot.
```

Comment:
I'm gonna use German here since this a german translation.

Unbegrenzte Zeichen- und Whiteboard-App hört sich irgendwie komisch an. So als ob es normal ein "Zeichenlimit" geben würde. Ich bin mir aber auch gerade nicht sicher wie man das am besten übersetzt. Vieleicht einfach "Infinite Whitboarding App". Das ist zwar kein deutsch, sollten die meisten aber verstehen.

Reply:
Alternativ einfach nur `Zeichen- und Whiteboard-App` oder `Freie Zeichen- und Whiteboard-App` (Frei im Sinne von kostenlos und unbegrenzt).